resolwe_bio/descriptors/general_clinical.yml

```diff
@@ -0,0 +1,1126 @@
+          required: false
+        - name: sampling
+          label: Sampling
+          type: basic:string
+          description: | 
+            Select which method was used to obtain the sample.
+          required: false
+          choices:
+            - label: Biopsy
+              value: biopsy
+            - label: Resection
+              value: resection
+        - name: sampling_type
+          label: Sampling type
+          type: basic:string
+          description: | 
+            Select where your sample was derived from, e.g. Primary (tumor) or Secondary (metastasis).
+          required: false
+          choices:
+            - label: Primary
+              value: primary
+            - label: Secondary
+              value: secondary
+        - name: biopsy_site
+          label: Biopsy site
+          type: basic:string
+          description: | 
+            General description of where your sample was derived from, e.g. stomach (primary site) or lymph nodes (secondary site), etc.
+          required: false
+        - name: grading
+          label: Grading
+          type: basic:string
+          description: | 
+            General description of the disease stage.
+          required: false
+        - name: disease_status
+          label: Disease status
+          type: basic:string
+          description: | 
+            Define status of the disease, e.g. progressive disease.
+          required: false
+        - name: disease_signature
+          label: Disease signature
+          type: basic:string
+          required: false
+        - name: metastases_sites
+          label: Metastases sites
+          type: basic:string
+          required: false
+        - name: number_of_metastases
+          label: Number of metastases
+          type: basic:integer
+          required: false
+        - name: mesenchymal
+          label: Mesenchymal
+          type: basic:string
+          description: | 
+            Select if the cells are of mesenchymal type.
+          required: false
+          choices:
+            - label: "Yes"
+              value: "yes"
+            - label: "No"
+              value: "no"
+        - name: date_of_progression
+          label: Date of progression
+          type: basic:date
+          required: false
+        - name: description_of_disease_information
+          label: Description of disease information
+          type: basic:string
+          required: false
+    - name: sample_preparation_and_processing
+      label: Sample preparation and processing
+      group:
+        - name: sample_obtained_in_site
+          label: Sample obtained in site
+          type: basic:string
+          required: false
+        - name: sample_type
+          label: Sample type
+          type: basic:string
+          description: | 
+            Source of biological material, e.g. fresh or FFPE.
+          required: false
+         - name: archival
```

Comment:
Incorrect indent. Should be aligned with `- name: sample_type`.

Reply:
this is not correct. Check default schema. It is the same